models/utils/int_zendesk__calendar_spine.sql

```diff
@@ -1,12 +1,13 @@
 -- depends_on: {{ source('zendesk', 'ticket') }}
 
 with spine as (
+    {% set current_ts = dbt.current_timestamp_backcompat() %}
```

Comment:
I believe current_ts can be set within the `{% if execute %}` block if you don't want to incur the cost of calling the macro during the parse step.

Reply:
@crericha thanks for the call out and I agree that it would be more performant to include in the execute block. Just tested on my end and all looks good with this adjustment!